package.json

```diff
@@ -96,6 +98,7 @@
     "chai-as-promised": "^7.1.1",
     "chai-exclude": "^2.1.0",
     "chai-shallow-deep-equal": "^1.4.6",
+    "chromedriver": "^90.0.1",
```

Comment:
I wonder if it would be better to leave this out of the package.json and just `npm install chromedriver@^90.0.1` somewhere in the npm run command. This would mean it only gets downloaded if you're actually running the chrome 90 test.